tests/secretCRUD_test.go

```diff
@@ -0,0 +1,162 @@
+package tests
+
+import (
+	"encoding/json"
+	"net/http"
+	"testing"
+	"time"
+
+	"github.com/openfaas/faas/gateway/requests"
+)
+
+const secretsURL = "http://127.0.0.1:31112/system/secrets"
```

Comment:
The `secretsURL` was fixed with [this commit](https://github.com/openfaas/certifier/pull/22/commits/45468fe29c91115e2b61b0ecb2e22a8725264947).